### PR TITLE
Port testable uniform session to a release branch

### PR DESCRIPTION
**Pull request by SzymonPobiega**
Add testable uniform session

### CODE REVIEW

File: src/NServiceBus.UniformSession.Testing/NServiceBus.UniformSession.Testing.csproj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <Description>NServiceBus Uniform Session Testing</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\UniformSession\NServiceBus.UniformSession.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Testing" Version="[8.0.0-alpha.155,9.0.0)" />
```
Comment (mikeminutillo) [inline src/NServiceBus.UniformSession.Testing/NServiceBus.UniformSession.Testing.csproj]: Should this be `[7.0.0,8.0.0)`?

File: src/NServiceBus.UniformSession.Testing/NServiceBus.UniformSession.Testing.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```
Comment (mikeminutillo) [inline src/NServiceBus.UniformSession.Testing/NServiceBus.UniformSession.Testing.csproj]: Should this be `netstandard2.0` for core v7?

**Review by mikeminutillo — APPROVED** [review]
I adjusted the package references and target frameworks to be in NServiceBus version 7 ranges.

**Comment by SzymonPobiega** [comment]
Good catch @mikeminutillo . Should be good now.

**Pull request merged**